### PR TITLE
start work on unbxd

### PR DESCRIPTION
**Pull request by hobadams**

### CODE REVIEW

File: docs/platform/integration/unbxed/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+id: getting-started
+title: Getting started
+sidebar_label: Getting started
+---
+
+## Before your start
+
+You'll need to have the offical Unbxed Magento module installed.
```
Comment (LesleyDuyndam) [inline docs/platform/integration/unbxed/getting-started.md:9]:
```suggestion
You'll need to have the offical Unbxd Magento module installed.
```

**Review by LesleyDuyndam — APPROVED** [review]

File: docs/platform/integration/unbxed/overview.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+id: overview
+title: Unbxed Integration
+sidebar_label: Overview
+---
+
+:::caution
+Unbxed is currently only supported with our Magento 2 shop backend.
```
Comment (LesleyDuyndam) [inline docs/platform/integration/unbxed/overview.md:8]:
```suggestion
Unbxd is currently only supported with our Magento 2 shop backend.
```

File: sidebars.js [hunk 3]
```diff
@@ -118,6 +118,14 @@ module.exports = {
           'platform/integration/contentful/content-models',
         ],
       },
+      {
+        type: 'category',
+        label: 'Unbxed',
```
Comment (LesleyDuyndam) [inline sidebars.js:123]:
```suggestion
        label: 'Unbxd',
```

**Comment by hobadams** [comment]
@mateuszwozniak @LesleyDuyndam - i've made changes based on the updated code. please review this when you get time.

**Review by LesleyDuyndam — APPROVED** [review]
Very clear, nice job! When unbxd patch is released i'll do a testrun based on this docs.

**Pull request merged**